js/ez-navigation-hub.js

```diff
@@ -1,10 +1,135 @@
 (function () {
     /** @polymerElement */
-    class EzNavigationHub extends Polymer.Element {
+    class NavigationHub extends Polymer.Element {
```

Comment:
and the doc of the element ? what kind of content it expects, is able to handle ? the fact that it needs `Element.matches` ?